### DISCUSSION TITLE
Integration OpenSTF with Katalon Studio

### DISCUSSION THREAD

**Discussion by brianrakhmat** [DeviceFarmer/stf#728] [Q&A]
Hi All, I want to use OpenSTF for testing a lot of real mobile device app, previously i already scripted in Katalon Studio and i want to integrate with OpenSTF for monitoring behaviour in mobile device, but until now i'm very confusing how integrate OpenSTF with Katalon Studio.

Anyone can help me for this ?

I'm highly appreciated for it.

**Comment by denis99999** [comment]
@brianrakhmat, I'm not familiar with Katalon Studio but the correct way to interact with DeviceFarmer/STF is through its RestFul API and ADB interface, all over the network

**Comment by brianrakhmat** [comment]
Hi, thanks for ur response. but it is possible or not if the test case i
triggered from Katalon Studio and then the STF showing the application
automate and also showing report testing from STF after finished run
testing?

**Comment by denis99999** [comment]
@brianrakhmat , I'm not sure I well understand you, but know that using STF through the API produces the same effects as through the UI, so for example yes we can see the mobile screen while running automatic test, etc. , simply log in to the UI using the same credentials used by the script that uses the API.

**Comment by brianrakhmat** [comment]
Hi Denis,

Hmm ok,
But is this Devicefarmer/STF is already support iOS ?

Thanks

On Tue, Nov 7, 2023 at 10:58 PM Denis Barbaron ***@***.***>
wrote:

> @brianrakhmat <https://github.com/brianrakhmat> , I'm not sure I well
> understand you, but know that using STF through the API produces the same
> effects as through the UI, so for example yes we can see the mobile screen
> while running automatic test, etc. , simply log in to the UI using the same
> credentials used by the script that uses the API.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DeviceFarmer/stf/discussions/728#discussioncomment-7500689>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEILCVHXWOWVPFDSXSNIHWLYDJLAZAVCNFSM6AAAAAA67KAGL2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKMBQGY4DS>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Salam,*
*Brian Rakhmat Aji*


*Contact me!*
*Ph & WA : +628 <//+62857-9741-4968>57-9741-4968 <//+62857-9741-4968>*
*Website <https://www.brianrakhmataji.com/> | Blog
<https://blog.brianrakhmataji.com/> | Facebook
<https://www.facebook.com/brianrakhmat> | Twitter
<https://twitter.com/brianrakhmat> | Linkedin
<https://www.linkedin.com/in/brianrakhmat>*

**Comment by denis99999** [comment]
> But is this Devicefarmer/STF is already support iOS ?

No
